### PR TITLE
Replacing paths in 'Requires' with package names.

### PR DESCRIPTION
**Pull request by PawelWMS**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

A small clean-up of our specs to unify our approach. This also removes the noisy `Translated '[path]' to package '[package_name]'` warnings from out toolkit's output and aligns us better with our future work.

###### Change Log  

- Replaced `Requires` on a specific path to a binary with the package name providing that binary.

###### Does this affect the toolchain?  

No.

###### Test Methodology

- None, minor clean-up.

### CODE REVIEW

File: SPECS/ca-certificates/ca-certificates.spec [hunk 1]
```diff
@@ -101,7 +101,6 @@ Source21:       certdata.base.txt
 Source22:       bundle2pem.sh
 Source23:       certdata.microsoft.txt
 
-BuildRequires:  /bin/ln
```
Comment (PawelWMS) [inline SPECS/ca-certificates/ca-certificates.spec:104]: Note: already provided by `coreutils` two lines below.

File: SPECS/ca-certificates/ca-certificates.spec [hunk 2]
```diff
@@ -71,7 +71,7 @@ Name:           ca-certificates
 
 # When updating, "Version" AND "Release" tags must be updated in the "prebuilt-ca-certificates" package as well.
 Version:        20200720
-Release:        13%{?dist}
+Release:        14%{?dist}
```
Comment (thcrain-msft) [inline SPECS/ca-certificates/ca-certificates.spec:74]: Please also bump prebuilt-ca-certificates as well

**Review by thcrain-msft — APPROVED** [review]
LGTM once prebuilt-ca-certificates is bumped

File: SPECS/ca-certificates/ca-certificates.spec (hunk 2 above)
Comment (PawelWMS) [reply SPECS/ca-certificates/ca-certificates.spec:74]: Thanks, done!

**Comment by PawelWMS** [comment]
Abandoning this PR. In some cases paths better reflect the spec's dependencies and replacing them with package name doesn't solve all our path-related build issues anyway.

**Pull request closed without merging**